frontend/src/components/postition-templates-list.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import PropTypes from "prop-types";
+import './components.css'
+
+export class PositionTemplatesList extends React.Component {
+    static propTypes = {
+        position_templates: PropTypes.arrayOf(
+            PropTypes.shape({
+                position_type: PropTypes.string,
+                offer_template: PropTypes.string
+            })
+        ).isRequired
+    };
+    render() {
+        const { position_templates } = this.props;
+        let templateList = <div>No Templates...</div>;
+        if (position_templates.length > 0) {
+            templateList = (
+                <ul>
+                    {position_templates.map(template => (
+                        <li key={template.offer_template}>
+                            <span className="positionTypeStyle">
```

Comment:
CSS names usually use `-` instead of camel case. Also, there is no need for `Style` because it is CSS, it's already clear that it's a style.  This whole thing should be namespaced. Add a `className="template-list"` to the surrounding `div`. Then, your CSS selector can be

```css
.template-list .position-type {
   ...
}
```